tests/store/test_footprints.py

```diff
@@ -1,10 +1,32 @@
 import pytest
+from functools import partial
 from helpers import get_footprint_datapath
 from openghg.retrieve import search
 from openghg.objectstore import get_bucket
-from openghg.store import Footprints, load_metastore
+from openghg.store import Footprints
 from openghg.util import hash_bytes
-from openghg.standardise import standardise_footprint
+from openghg.standardise._standardise import standardise_footprint, standardise_from_binary_data
+
+@pytest.fixture(autouse=True, scope="module")
+def auto_bucket():
+    global bucket
+    bucket = get_bucket()
+    yield bucket
+    globals().pop('bucket')
+
+
+@pytest.fixture(autouse=True)
+def standardise():
+    """Use `standardise(**kwargs)` to standardise footprints data.
+
+    We only deal with footprints data in this test module, and most
+    tests just use `bucket = get_bucket()`, so this works for
+    most tests in this module.
+
+    This was added to help refactor `BaseStore` and its children
+    by removing uses of the context manager for using `read_file`.`
+    """
+    return partial(standardise_footprint, bucket=bucket)
```

Comment:
Again, has this been created as a means to reduce repetition for creating the bucket?

Reply:
Yes, I didn't figure out how to make an autouse fixture set a variable (e.g. bucket) until after I had updated these tests. And originally the `standardise` fixture did a lot more, because I didn't update the standardise_* functions at first, then later decided it made sense to do that.